src/backend/src/pct.jl

```diff
@@ -141,6 +140,7 @@ function process_pct_geometry(dir="$(@__DIR__)/../data/")
 
 
     # Save new geojson and generate tiles from it
+    # Should this be a temp directory? do we use the geojson for anything else?
```

Comment:
TODO: look into this

Reply:
We use the geojson for getting the zone centroids, though we could pre-process those.

I don't think I've noted it anywhere yet, but the zone centroids code has some data quality issues: there end up being more centroids than zones. Should probably fix that sometime.